### PR TITLE
Implement basic dropdown cell and article cell

### PR DESCRIPTION
**Pull request by jassmith**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,8 +34,8 @@
         "@types/faker": "^5.5.8",
         "@types/lodash": "^4.14.177",
         "@types/marked": "^4.0.2",
-        "@types/react": "^16.14.21",
-        "@types/react-dom": "^16.9.5",
+        "@types/react": "^16.14.21 || 17.x",
+        "@types/react-dom": "^16.9.5 || 17.x",
```
Comment (ivoelbert) [inline package.json:38]: woooo fancy

File: packages/cells/src/cells/dropdown-cell.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+interface CustomMenuProps extends MenuProps<any> {}
+
+export class CustomMenu extends React.Component<CustomMenuProps> {
+    render() {
+        const { Menu } = components;
+        const { children, ...props } = this.props;
+        return <Menu {...props}>{children}</Menu>;
+    }
+}
```
Comment (ivoelbert) [inline packages/cells/src/cells/dropdown-cell.tsx:15]: Why is this a class, lol

File: packages/cells/src/cells/dropdown-cell.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+export type DropdownCell = CustomCell<DropdownCellProps>;
+
+const Wrap = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+`;
+
+const editor: ProvideEditorCallback<DropdownCell> = () => {
```
Comment (ivoelbert) [inline packages/cells/src/cells/dropdown-cell.tsx:32]: I'm pretty sure you can do
```ts
const InnerEditor: ReturnType<ProvideEditorCallback<DropdownCell>> = p => {
    // ...
}

InnerEditor.disablePadding = true

const editor: ProvideEditorCallback<DropdownCell> = () => InnerEditor;
```

This way `editor` will now return a stable reference all the time, and you won't have to ignore `rules-of-hooks`

**Review by ivoelbert — COMMENTED** [review]
I think this alternative is reasonable

File: packages/cells/src/cells/dropdown-cell.tsx (hunk 2 above)
Comment (jassmith) [reply packages/cells/src/cells/dropdown-cell.tsx:15]: Followed an example blindly.

Comment (jassmith) [reply packages/cells/src/cells/dropdown-cell.tsx]: Fixed

File: packages/cells/src/cells/dropdown-cell.tsx (hunk 3 above)
Comment (jassmith) [reply packages/cells/src/cells/dropdown-cell.tsx]: You are right, fixed

**Pull request merged**